### PR TITLE
Make the measurement duration configurable

### PR DESCRIPTION
**Pull request by tdabasinskas**
PR #3 introduced a static `30s` duration to repeat measurements for in case the current time drift is above `10ms`. Anyhow, `30s` duration does not work well in case when we have Prometheus scrape timeout configured to a smaller number (e.g. `10s`). In this case, the scrape timeouts, without getting results from the exporter.

This PR allows to override (still keeping `30s` as default to avoid breaking existing setups) this measurement duration, allowing to reduce it to align it with Prometheus scrape timeout.

I've named the setting `--ntp.measurement-duration`, but please let me know if you think there's a better name for it 🙂 

I've also added additional `warning` log message in case the drift was to high and multiple measurements were requires, so it's easier to indicate that's the case and know the exact values of `high drift` and `measurement duration`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,22 +25,23 @@ make GOLDFLAGS="-w -linkmode external -extldflags -static" && docker build .
 
 ## Usage
 
-Command-line options: (Only `-ntp.server` is required.)
+Command-line options: (Only `--ntp.server` is required.)
```
Comment (majewsky) [inline README.md]: Package `flag` uses single dashes for command-line options. Why did you change to double dashes throughout this file?

Comment (tdabasinskas) [reply README.md]: Hi,

Actually [`flags` documentation](https://golang.org/pkg/flag/) states that both single and double dashes are supported.

It's usually a good practice to use single dash for short argument names and double dashes for full argument names (Prometheus [does the same](https://prometheus.io/docs/prometheus/latest/configuration/configuration/)).

So, basically it's just a documentation change, but if you would prefer keeping it as is, I can revert it.

Comment (majewsky) [reply README.md]: > It's usually a good practice to use single dash for short parameter names and double dashes for full parameter names

I'm on board with that as well, but thought the convention is to prefer single dashes when using `flag`. See for example `go help build`, where long options also use single dashes.

I'd prefer if you revert that particular documentation change to avoid the appearance that upgrading users need to change their invocations.

Comment (tdabasinskas) [reply README.md]: Done!

**Comment by tdabasinskas** [comment]
Thanks for merging this, @majewsky. Any plans to do a new Release?

**Comment by majewsky** [comment]
@tdabasinskas At long last: https://github.com/sapcc/ntp_exporter/releases/v1.1.0
